multicluster/controllers/multicluster/member/gateway_controller.go

```diff
@@ -29,7 +29,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller"
```

Comment:
Could you point to me where we get ClusterSet event and map it to other objects? And how we filter and ingore ClusterSet updates?

Reply:
But we need not to watch ClusterSets in gateway_controller.go too?

Reply:
No need to watch ClusterSets in gateway_controller.go, node_controller.go is responsible to create the Gateway, gateway_controller.go will do its job after the Gateway is created.